setup.py

```diff
@@ -25,7 +25,7 @@
         "Programming Language :: Python :: 3.8",
     ],
     packages=find_namespace_packages(exclude=["tests"]),  # Required
-    install_requires=["bioimageio.spec", "imageio>=2.5", "numpy", "ruamel.yaml", "tqdm", "xarray"],
+    install_requires=["bioimageio.spec>=0.3.3post7", "imageio>=2.5", "numpy", "ruamel.yaml", "tqdm", "xarray"],
```

Comment:
even with this minimal bound an older bioimageio.spec version seems to be installed in the GH actions workflow... I suppose the 'post' doesn't work in the comparison. going to bump bioimageio.spec patch instead...

Reply:
No, it works, but the version is not available via conda yet, see the error log:
```
'bioimageio.spec>=0.3.3post7' distribution was not found and is required by bioimageio.core
```

Reply:
Note that it can take a few hours till new versions on conda-forge become available, because they need to be synched with some mirror. So just retry this tomorrow and it should work.

Reply:
I had manually checked here: https://anaconda.org/conda-forge/bioimageio.spec/files and post7 was listed. But I think I looked again at the old error somehow...

Reply:
> I had manually checked here: https://anaconda.org/conda-forge/bioimageio.spec/files and post7 was listed.

It can still take up to a few hours before the package becomes available after being listed on anaconda.org, because it must be synced with some CDN.

Reply:
it does, however now conda build is not having it:
`conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {"bioimageio.spec[version='>=0.3.3post7']"}
`
???

Reply:
Ok, apparently it can't resolve the `post` stuff with `>=`. Just remove the post and it should pull in the latest version by default:
```
(main) pape@mokshi:~$ conda install -c conda-forge bioimageio.spec
Collecting package metadata (current_repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /home/pape/Work/software/conda/miniconda3/envs/main

  added / updated specs:
    - bioimageio.spec


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    bioimageio.spec-0.3.3.post7|     pyhd8ed1ab_0          66 KB  conda-forge
```